_pages/events.md

```diff
@@ -5,24 +5,30 @@ image: images/gen.jpg
 permalink: /about/events/index.html
 ---
 
-## Conferences  
+## Conferences
 
-_The Color of Surveillance. April, 2016_  
-Hosted by by Georgetown Law, the [conference](https://www.law.georgetown.edu/academics/centers-institutes/privacy-technology/events/index.cfm) focused on the policing and surveillance of black communities. Freddy Martinez, Director of Lucy Parsons Labs, gave a primer on Stingray devices, also known as IMSI catchers, and their role in the monitoring of the black community.
+### Surveillance Studies Network / Surveillance & Society biannual conference (June 7-9, 2018)
```

Comment:
Mixed casing here. Also, it is a nit here but we use various formats for the dates so let's be consistent and just say (Month, YEAR)